daemon/mgr/spec_mount.go

```diff
@@ -14,6 +14,21 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+const (
+	// PropagationRPrivate represents mount propagation rprivate.
+	PropagationRPrivate = "rprivate"
```

Comment:
How about changing all this const variables' naming to be XXXXXPropagationMode?
I think this will be a little bit more readable.